sycl/source/detail/program_manager/program_manager.cpp

```diff
@@ -487,20 +487,28 @@ string_class ProgramManager::getProgramBuildLog(const RT::PiProgram &Program,
   string_class Log = "The program was built for " +
                      std::to_string(PIDevices.size()) + " devices";
   for (RT::PiDevice &Device : PIDevices) {
+    std::string DeviceBuildInfoString;
```

Comment:
The same Size variable is used in 3 different contexts. Can we please have different var per context, since the code modified anyway? E.g. ProgramInfoSize, BuildInfoSize, DeviceInfoSize

Reply:
done